src/test/resources/authz.json

```diff
@@ -37,6 +37,13 @@
         "m2m"
       ],
       "route": "getAttributeByOriginAndCode"
+    },
+    {
+      "verb": "GET",
```

Comment:
Strangely the test is working fine even if the verb here is wrong as you pointed out

Reply:
We reviewed the tests with @TonioGela and we verified that these tests are unnecessary because the check on the `verb` is implemented on the class `AttributeApi` that is automatically generated from the code generation.

In this case we can probaly remove these verbs inside thae `authz.json.`